invokers/python/pyfunc/__main__.py

```diff
@@ -2,31 +2,36 @@
 # SPDX-License-Identifier: BSD-2-Clause
 
 from pyfunc import invoke, find_func
+from pyfunc.constants import *
 
 import sys
 from argparse import ArgumentParser
 
 
 def check(args):
     try:
-      find_func(args.search_path)
+      find_func(args.search_path, args.module_name, args.function_name)
     except Exception as e:
       print("Failed to detect function: " + str(e), file=sys.stderr)
       sys.exit(1)
 
 def start(args):
-    invoke.main(args.search_path)
+    invoke.main(args.search_path, args.module_name, args.function_name)
 
 parser = ArgumentParser(prog='pyfunc')
 parser.set_defaults(func=lambda: parser.print_help())
 subparsers = parser.add_subparsers(help='sub-command help')
 
 parser_check = subparsers.add_parser('check', help='check if the module and function can be loaded')
 parser_check.add_argument('-s', '--search_path', type=str, default='.')
+parser_check.add_argument('-m', '--module_name', type=str, default=MODULE_NAME_DEFAULT)
```

Comment:
Default values for the path, module, and function are now only here, instead of spread out in various places in the code (i.e. as default function parameter values). This single source of truth for the defaults should make it a little easier to follow.